Library/Tracking/KSRAnalyticsTests.swift

```diff
@@ -1314,6 +1326,31 @@ final class KSRAnalyticsTests: TestCase {
     XCTAssertNil(self.segmentTrackingClient.traits)
   }
 
+  // MARK: CTAs Tracking
```

Comment:
I think the MARK is a good idea if we are putting all `CTA Clicked` events under here but right now I think they are all scattered. Two options imo:
1. Put all the existing CTA Clicked tests in here
2. Remove the MARK